app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Button, TextField, Tooltip, withStyles } from '@material-ui/core';
+import IconSVG from 'components/shared/IconSVG';
+import styled from 'styled-components';
+const colors = require('styles/colors.scss');
+const errorRed = colors.red03;
+const blueType = '#397cf1';
+
+export const EllipsisText = styled.div`
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+`;
+
+export const PipelineName = styled(EllipsisText)`
+  font-size: 14px;
+  margin-top: 5px;
+  line-height: 17px;
+
+  ${({ errorName }) => errorName && `color: ${errorRed};`}
+`;
+
+export const PipelineDescription = styled(EllipsisText)`
+  color: gray;
+  font-size: 12px;
+`;
+
+export const CustomTooltip = withStyles(() => {
+  return {
+    tooltip: {
+      fontSize: '13px',
+    },
+  };
+})(Tooltip);
+
+export const ErrorTooltip = withStyles(() => {
+  return {
+    arrow: {
+      color: errorRed,
+    },
+    tooltip: {
+      background: errorRed,
+    },
+  };
+})(CustomTooltip);
+
+export const SaveButton = styled(Button)`
```

Comment:
Per our discussion earlier, let's keep the standard Mui button styling and use the shared components. After converting, we can restyle them how we choose.